src/Microsoft.Health.Dicom.Functions/Indexing/Configuration/ReindexConfiguration.cs

```diff
@@ -39,5 +41,13 @@ public class ReindexConfiguration
         ///// </summary>
         //[Range(0d, 1d)]
         //public double MaxErrorPercentage { get; set; } = 1;
+
+        public bool IsPurgeErrorEnabled { get; set; } = true;
+
+        public IEnumerable<OrchestrationRuntimeStatus> RuntimeStatus { get; } = new[] { OrchestrationRuntimeStatus.Completed };
+
+        public DateTime CreatedTimeFrom { get; set; } = DateTime.UtcNow.Subtract(TimeSpan.FromDays(7));
```

Comment:
The configuration is resolved upon starting the function app -- not at the time of starting each function invocation. So something like the start and end times cannot be present here. Instead, you could have perhaps a "minimum age" for deletion, like 2 weeks or a month.